ctrl/cleanUP

```diff
@@ -18,6 +18,7 @@ kill_named_process em100
 kill_named_process ttyd
 kill_named_process ttyd
 kill_named_process ttyd
+kill_named_process ttyd
```

Comment:
Why did we had to add it ?

Reply:
Starting a contest job is adding one more ttyd process. So we need to call "kill_named_process" to kill the 4th ttyd process.